pymc/data.py

```diff
@@ -92,6 +92,9 @@ class MinibatchIndexRV(IntegersRV):
 class MinibatchOp(OpFromGraph):
     """Encapsulate Minibatch random draws in an opaque OFG."""
 
+    # FIXME: __props__ should not be empty
+    __props__ = ()
```

Comment:
The underlying issue is that OpFromGraph doesn't have equality implemented: https://github.com/pymc-devs/pytensor/issues/1606

Reply:
sure